recipes/example/meta.yaml

```diff
@@ -1,34 +1,44 @@
-{% set version = "1.0" %}
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "3.8.2" %}
 
 package:
-  name: example
-  version: {{ version }}
+    name: simplejson
+    version: {{ version }}
 
 source:
-  git_url: https://github.com/conda-forge/staged-recipes.git
-  git_tag: 24507f6c6b2d2e03caee550e13830df15d791751
+    # It is preferable to get the package "closer" to the source. For example use GitHub instead PyPI whenever possible.
```

Comment:
We decided recently we can use either. We just shouldn't be cloning. So, the tarball is fine.